### PR TITLE
Added Project Euler

### PR DESCRIPTION
**Pull request by shubhamsah**
Added a folder named Project Euler that contains solutions to Project Euler Problems

### CODE REVIEW

File: Project Euler/problem_01/sol1.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"""
+Problem Statement:
+If we list all the natural numbers below 10 that are multiples of 3 or 5,
+we get 3,5,6 and 9. The sum of these multiples is 23.
+Find the sum of all the multiples of 3 or 5 below N.
+"""
+"""
```
Comment (Rishit-dagli) [inline Project Euler/problem_01/sol1.py:7]: Not a huge edit but I see you have been following a consistent style, could you remove the double `"""`

**Review by Rishit-dagli — CHANGES_REQUESTED** [review]
I am still performing a content review on this, but what is your take about having the following directory structure instead-

```
Project Euler
├── Problem 1
|   └── README.md (The problem statement)
|   └── solution1.py
|   └── solution2.py
├── Problem 2
|   └── README.md (The problem statement)
|   └── solution1.py
|   └── solution2.py
├── Problem ...
└── README.md (A bit about Project Euler)
```

**Comment by shubhamsah** [comment]
Alright, will do the necessary changes

**Comment by shubhamsah** [comment]
All the necessary changes requested has been done

**Review by Rishit-dagli — APPROVED** [review]
This looks good to me, I will go ahead and merge this. I have also made minor edits in README styling via [`502bde0`](https://github.com/Rishit-dagli/Python3-by-practice/pull/49/commits/502bde0a0ecec27515d1d89915e5134fb81e1415) and [`a995fd3`](https://github.com/Rishit-dagli/Python3-by-practice/pull/49/commits/a995fd330fd85ffaf8e5715d07c66214d4cc4fbe). Thank you for your contribution 💯 Feel free to support this repo by leaving a ⭐ !

**Comment by Rishit-dagli** [comment]
@all-contributors please add @shubhamsah for code

**Pull request merged**